### PR TITLE
Add percona-server56

### PR DESCRIPTION
**Pull request by magneland**
This is a straight copy of
https://raw.githubusercontent.com/Homebrew/homebrew-core/367439579ece4ae429e56fc3fd24ae7b36ecbca0/Formula/percona-server.rb

### CODE REVIEW

**Comment by magneland** [comment]
The homebrew-core formula recently got bumped to 5.7

**Comment by magneland** [comment]
Actually I will amend this a bit...

File: percona-server56.rb [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  url "https://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.28-76.1/source/tarball/percona-server-5.6.28-76.1.tar.gz"
+  version "5.6.28-76.1"
+  sha256 "ab8ab794a58a82132645ae84b74de91c7f9a5bcf81f2162628ce8976a00a4fd4"
+
+  bottle do
+    revision 1
+    sha256 "b283468128a1450e20c73ceb16f5d6454a2cb834b5f7b889118456d6f9693af6" => :el_capitan
+    sha256 "56b11a60d823385bbe3f888d43d2780fbe7ae0ac96745ea624095480b96d0602" => :yosemite
+    sha256 "aa6efe7ebbcdfa1a26530300da885f56a33d3132f953886cdbe309fb545da6ec" => :mavericks
+  end
```
Comment (mikemcquaid) [inline percona-server56.rb]: You can remove this bottle block as it's a new formula.

File: percona-server56.rb [hunk 2]
```diff
@@ -0,0 +1,193 @@
+class PerconaServer < Formula
```
Comment (mikemcquaid) [inline percona-server56.rb]: This needs named `PerconaServer56`

File: percona-server56.rb [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+  fails_with :llvm do
+    build 2334
+    cause "https://github.com/Homebrew/homebrew/issues/issue/144"
+  end
+
+  # Where the database files should be located. Existing installs have them
+  # under var/percona, but going forward they will be under var/msyql to be
+  # shared with the mysql and mariadb formulae.
+  def datadir
+    @datadir ||= (var/"percona").directory? ? var/"percona" : var/"mysql"
```
Comment (mikemcquaid) [inline percona-server56.rb]: You may want to consider making this data directory version dependent e.g. append a `56`?

File: percona-server56.rb (hunk 2 above)
Comment (magneland) [inline percona-server56.rb]: Ah thx.

File: percona-server56.rb (hunk 1 above)
Comment (magneland) [inline percona-server56.rb]: OK thx

File: percona-server56.rb (hunk 3 above)
Comment (magneland) [inline percona-server56.rb]: I guess the logic here (which I copied from the original formula) is that it's vendor-agnostic as well as version-agnostic. I don't know if there's any advantages to that. I can change it if you think it makes sense.

File: percona-server56.rb [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+      -DWITHOUT_AUTH_PAM=1
+      -DWITHOUT_AUTH_PAM_COMPAT=1
+      -DWITHOUT_DIALOG=1
+    ]
+
+    # TokuDB is broken on MacOsX
+    # https://bugs.launchpad.net/percona-server/+bug/1531446
+    args.concat %W[-DWITHOUT_TOKUDB=1]
+
+    if build.devel?
```
Comment (DomT4) [inline percona-server56.rb]: Can kill this block. There's no `devel` here.

File: percona-server56.rb [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+      s.gsub!(/^(PATH=".*)(")/, "\\1:#{HOMEBREW_PREFIX}/bin\\2")
+      # pidof can be replaced with pgrep from proctools on Mountain Lion
+      s.gsub!(/pidof/, "pgrep") if MacOS.version >= :mountain_lion
+    end
+
+    bin.install_symlink prefix/"support-files/mysql.server"
+
+    # mysqlaccess deprecated on 5.6.17, and removed in 5.7.4.
+    # See: https://bugs.mysql.com/bug.php?id=69012
+    if build.stable?
```
Comment (DomT4) [inline percona-server56.rb]: We'll always be building stable here, so you can remove this block _but_ keep the contents of what is currently in it, just fix the indentation.

File: percona-server56.rb (hunk 4 above)
Comment (magneland) [inline percona-server56.rb]: OK thx

File: percona-server56.rb (hunk 5 above)
Comment (magneland) [inline percona-server56.rb]: OK thx

**Comment by magneland** [comment]
I'm curious about how the brew-bot works? Does it run after every commit?

**Comment by DomT4** [comment]
Yup!

**Comment by magneland** [comment]
OK I believe this should be good now.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/bc95d6822bc2d22312a49681cf052c52677614d9. Thank you for your contribution to Homebrew @magneland; we appreciate it! :smiley_cat:

**Pull request closed without merging**